### PR TITLE
Update README.md for mounting on mac 

### PR DESCRIPTION
**Pull request by blundell**
screen shot 2017-05-14 at 9 23 19 pm

add the -t command which is required on mac

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,4 +101,4 @@ the License.
 [dev-preview-download]: https://dl.google.com/dl/androidthings/rpi3/devpreview/3.1/androidthings_rpi3_devpreview_3_1.zip
 [set-activity-controls]: https://developers.google.com/assistant/sdk/prototype/getting-started-other-platforms/config-dev-project-and-account#set-activity-controls
 [mic]: https://www.adafruit.com/product/3367
-[speaker]: https://www.adafruit.com/product/3369
\ No newline at end of file
+[speaker]: https://www.adafruit.com/product/3369
```
Comment (blundell) [inline README.md]: looks like github automatically added or removed the new line at the end of the file when I edited inline in the browser

File: README.md [hunk 2]
```diff
@@ -13,10 +13,18 @@ Pre-requisites
...
+        
+              Linux:
 
               # replace sdb1 with the sdcard reader device.
               mount /dev/sdb1 /mnt/disk
 
+              Mac:
+        
+              # replace disk2s1 with the sdcard reader device.
+              mount -t msdos /dev/disk2s1 /mnt/disk
```
Comment (proppy) [inline README.md]: Since this work on linux too, maybe we could merge the two with `-t`.

Comment (blundell) [inline README.md]: sure thing! https://github.com/androidthings/sample-googleassistant/pull/12/files#diff-04c6e90faac2675aa89e2176d2eec7d8

**Comment by proppy** [comment]
Thanks!

**Pull request merged**